### ISSUE TITLE
Training with saved weights

### ISSUE THREAD

**Issue by AdamJochna** [kentaroy47/frcnn-from-scratch-with-keras#12]
Hello,
I trained model for 10h and training is completed model is saved into model_frcnn.hdf5 and I have config.pickle. I would like to start training next time using this model not pretrained resnet50_weights_tf_dim_ordering_tf_kernels.h5 but it seems that if I choose model_frcnn.hdf5 as input weights in next training weights are not loaded because loss is much higher than at the end of last training. Can I do that somehow? I don't want to train every time from beggining using pretrained resnet50 but my trained model_frcnn.hdf5

**Comment by kentaroy47** [comment]
same issue as in #13?
let me look into this.

**Comment by kentaroy47** [comment]
fixed and confirmed that it works.. :) note that I tested with mobilenet.

now you can specify what model to load with --load
look at readme on test_frcnn.py.

**Comment by kentaroy47** [comment]
now supports resumed training

```
python train_frcnn.py --network mobilenetv1 -o pascal_voc -p ./VOCmydata/ --elen 180 --load model_frcnn.hdf5 
Using TensorFlow backend.
Parsing annotation files
Training images per class:
{'bg': 0, 'person': 151}
Num classes (including bg) = 2
Config has been written to config.pickle, and can be loaded when testing to ensure correct results
Num train samples 91
Num val samples 0
loading weights from ./pretrain/mobilenet_1_0_224_tf.h5
loading previous model from  model_frcnn.hdf5
Starting training
Epoch 1/50
 26/180 [===>..........................] - ETA: 3:34 - rpn_cls: 1.3765 - rpn_regr: 0.0025 - detector_cls: 0.0383 -
```

**Comment by AdamJochna** [comment]
Thanks a lot !!
